R/data.R

```diff
@@ -0,0 +1,14 @@
+#' List of all available styles
+#'
+#' A list containing all available styling options for hierplane
+#'
+#' Source: \url{https://github.com/allenai/hierplane/}
+"styles"
+
+#' Starwars Films and Starships
+#'
+#' A sample hierplane dataframe derived from the {dplyr} `starwars` dataset that
+#' shows the appearances of each starship in the first seven Star Wars movies.
+#'
+#' Source: \url{https://dplyr.tidyverse.org/reference/starwars.html}
+"hp_starwars"
```

Comment:
Oooo, what if we ditch the `hp_` and just call this `starships`?

Reply:
SMART

Reply:
hahahahahaha 😆